customfiles/depenguin_bsdinstall.sh

```diff
@@ -11,7 +11,7 @@ exit_error() {
 
 # read in variables
 if [ -f depenguin_settings.sh ]; then
-	# shellcheck source=sample.depenguin_settings.sh
+	# shellcheck source=customfiles/depenguin_settings.sh.sample
```

Comment:
This was important - shellcheck includes files relative to the current working directory from where it was invoked. So when this was simply "depenguin_settings.sh.sample", it wouldn't find the file and cause you all the trouble you've seen.

Reply:
@bretton This ☝️☝️☝️☝️☝️